### ISSUE TITLE
Suggestion: Add a screenshot of the web interface to the README

### ISSUE THREAD

**Issue by sbrl** [SkullTech/journalist#2]
Adding a screenshot of the web interface to the README will help people to see what it's like before they download it to try it out.

**Comment by skulltech** [comment]
Yeah, I should've done it a long time ago. Thanks for reminding me of it.
